### PR TITLE
Bug fix: render Markdown-code link text as code (#620)

### PR DESCRIPTION
**Pull request by egnha**

### CODE REVIEW

**Comment by hadley** [comment]
@gaborcsardi is this good to merge?

**Comment by gaborcsardi** [comment]
@hadley yes.

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # roxygen2 6.0.1.9000
 
+## Minor features and bug fixes
```
Comment (hadley) [inline NEWS.md]: Can you please remove this heading? We add news headings just prior to release.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**